src/parseRuntimeAndValidateSDK.ts

```diff
@@ -57,15 +57,19 @@ export function getHumanFriendlyRuntimeName(runtime: string): string {
  * package.json.
  * @param sourceDir directory where the functions are defined.
  */
-export function getRuntimeChoice(sourceDir: string): any {
+export function getRuntimeChoice(sourceDir: string): string {
   const packageJsonPath = path.join(sourceDir, "package.json");
   const loaded = cjson.load(packageJsonPath);
   const engines = loaded.engines;
   if (!engines || !engines.node) {
-    return null;
-    // TODO(b/129422952): Change to throw error instead of returning null
-    // when engines field in package.json becomes required:
-    // throw new FirebaseError(ENGINES_FIELD_REQUIRED_MSG, { exit: 1 });
+    // We should really never hit this, since deploy/functions/prepare already checked that package.json has an "engines" field.
+    throw new FirebaseError(
+      `Engines field is required but was not found in package.json.\n` +
+        `To fix this, add the following lines to your package.json: \n
+      "engines": {
+        "node": "10"
```

Comment:
More of a product question, but do we want to tell users to go to Node 10?

Reply:
That's in the other PR (Michael's)

Reply:
@mbleigh for input. I think is, because Node 10 is going to GA quite soon, so it's a good default to suggest. (And those that really care about only using GA runtimes will not have an empty engine field in package.json. )

Reply:
To capture our offline conversation: the init templates will stay at Node 8, but we can update the error messages now.